src/app/portlets/dot-service-integration/dot-service-integration-configuration/dot-service-integration-configuration.component.html

```diff
@@ -0,0 +1,78 @@
+<div class="dot-service-integration-configuration__container">
+    <div class="dot-service-integration-configuration__header">
+        <dot-avatar
+            [size]="112"
+            [url]="serviceIntegration.iconUrl"
+            [label]="serviceIntegration.name"
+        ></dot-avatar>
+        <div class="dot-service-integration-configuration__data">
+            <span class="dot-service-integration-configuration__service-name">
+                {{ serviceIntegration.name }}
+            </span>
+            <div class="dot-service-integration-configuration__service-key">
+                {{ messagesKey['service.integration.key'] }}
+                <dot-copy-button
+                    [copy]="serviceIntegration.key"
+                    [label]="serviceIntegration.key"
+                ></dot-copy-button>
+            </div>
+            <span class="dot-service-integration-configuration__configurations"
+                >{{
+                    serviceIntegration.configurationsCount
```

Comment:
Isn't this the same as the other PR? if is, should we just move this its own component and reuse in all the portlet?

Reply:
no it's different